bitcoind2.Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM kylemanna/bitcoind:latest
```

Comment:
The only change between `bitcoind1.Dockerfile` and `bitcoind2.Dockerfile` is the `CMD` line. I suggest you create a single file, `bitcoind.Dockerfile`, with the following content

```
FROM kylemanna/bitcoind:latest
COPY start-bitcoind.sh /usr/local/bin/start-bitcoind.sh
RUN chmod +x /usr/local/bin/start-bitcoind.sh
ENTRYPOINT ["/usr/local/bin/start-bitcoind.sh"]
```

and then it can be run as

```
docker run bitcoind 31591 https://localhost:31592 0.0.0.0:32591 0.0.0.0/0 admin admin
```

for `bitcoind1` and

```
docker run bitcoind 31592 https://localhost:31591 0.0.0.0:32592 0.0.0.0/0 admin admin
```

for `bitcoind2`.

Reply:
the only reason for these is to be able to build and push in CI https://github.com/jurajpiar/rskj/actions/runs/9059773191/workflow#L149-L170

Reply:
I wasn't aware that GHA doesn't allow to set parameters for service containers. 🙄 

Anyways, you can still do with a single Dockerfile. In fact, you can use directly the `kylemanna/bitcoind` image (although we should migrate at something selfhosted at some point) and then leverage volumes for the service ( https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idservicesservice_idvolumes ) to put the right config file for each container. https://github.com/kylemanna/docker-bitcoind/blob/master/docs/config.md shows where the config file should be mounted to be picked up by `bitcoind` in this container.

Reply:
yeap, tried that...